docs/monitoring/05-01-alertmanager.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Alertmanager sub-chart
```

Comment:
WHy it is relevant that there is a sub-chart? Isn't that confusing for the end-user? WHat value does he have from that info?

Reply:
We are following the template here and we distinguish charts from the sub-charts for the users to easily locate those values.

Reply:
ok